src/main/java/picard/fingerprint/CrosscheckReadGroupFingerprints.java

```diff
@@ -26,104 +26,100 @@
+ * Program to check that all (read-)groups within the set of BAM files appear to come from the same
+ * individual. Can br used to cross-check libraries or samples too.
  *
  * @author Tim Fennell
+ * @author Yossi Farjoun
  */
 @CommandLineProgramProperties(
-        usage = "Checks if all read groups within a set of BAM files appear to come from the same individual",
-        usageShort = "Checks if all read groups appear to come from the same individual",
+        usage = "Checks if all (read-)groups within a set of BAM files appear to come from the same individual.",
+        usageShort = "Checks if all (read-)groups appear to come from the same individual.",
         programGroup = Fingerprinting.class
 )
 public class CrosscheckReadGroupFingerprints extends CommandLineProgram {
 
-    @Option(shortName= StandardOptionDefinitions.INPUT_SHORT_NAME,
-            doc="One or more input BAM files (or lists of BAM files) to compare fingerprints for.")
+    @Option(shortName = StandardOptionDefinitions.INPUT_SHORT_NAME,
+            doc = "One or more input BAM files (or lists of BAM files) to compare fingerprints for.")
     public List<File> INPUT;
 
-    @Option(shortName=StandardOptionDefinitions.OUTPUT_SHORT_NAME, optional=true,
-            doc="Optional output file to write metrics to. Default is to write to stdout.")
+    @Option(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, optional = true,
+            doc = "Optional output file to write metrics to. Default is to write to stdout.")
     public File OUTPUT;
 
-    @Option(shortName="H", doc="The file of haplotype data to use to pick SNPs to fingerprint")
+    @Option(shortName = "H", doc = "The file of haplotype data to use to pick SNPs to fingerprint")
     public File HAPLOTYPE_MAP;
 
-    @Option(shortName="LOD",
-            doc="If any two read groups match with a LOD score lower than the threshold the program will exit " +
-                "with a non-zero code to indicate error. 0 means equal probability the read groups match vs. " +
-                "come from different individuals, negative numbers mean N logs more likely that the read groups " +
-                "are from different individuals and positive numbers mean N logs more likely that the read groups " +
-                "are from the sample individual.")
+    @Option(shortName = "LOD",
+            doc = "If any two groups (with the same sample name) match with a LOD score lower than the threshold " +
+                    "the program will exit with a non-zero code to indicate error." +
+                    " Program will also exit with an error if it finds two groups with different sample name that" +
+                    "match with a LOD score greater than -LOD_THRESHOLD." +
+                    "\n\n" +
+                    "LOD score 0 means equal likelihood" +
+                    "that the groups match vs. come from different individuals, negative LOD scores mean N logs more likely " +
```

Comment:
Need a space at the beginning of this string (or at the end of the string on the line above)